src/scss/_colors.scss

```diff
@@ -1,11 +1,22 @@
 $white: #fff;
-$white1: #bcdbee;
-$black_0-3: #0000004d;
+$bluish-white: #bcdbee;
+
+$black-light-tint: #0000004d;
+$black-dark-tint: #000000de;
```

Comment:
covert this to hexadecimal

Reply:
in `#0000004d` there is a black color `#000000` which has a opacity of 0.3 represented by `4d`

so if I take its solid color with no opacity... then its meaningless. see the footer its a balck tint over the footer.

Reply:
ok